libs/runtime/ebpf_epoch.c

```diff
@@ -1028,3 +1128,89 @@ _ebpf_epoch_work_item_callback(_In_ cxplat_preemptible_work_item_t* preemptible_
 
     cxplat_release_rundown_protection(&_ebpf_epoch_work_item_rundown_ref);
 }
+
+/**
+ * @brief Add the CPU to the next active CPU table. Must be called when running on cpu_id.
+ *
+ * @param[in] cpu_id CPU to add.
+ */
```

Comment:
This seems odd to say it must be called on the cpu passed as an argument, when the caller or callee could just call ebpf_get_current_cpu() making the argument redundant.  Why do we need an argument at all?  (If there is a reason, the comments here should explain it.)

Reply:
Comments added. Short version is that ebpf_get_current_cpu costs ~5ns and I the code should try to avoid requerying it.